### PR TITLE
Fix date bugs in matrix functionality

### LINKED ISSUE #141: Issues with matrix, unable to open app
- When there is no tasks in json, there will be a NoSuchElement error that prevents app opening.
- When all tasks in json have due date before current date, there will be divided by zero error (split = 0 in MainWindow.java) that prevents app opening.

### PR DESCRIPTION
**Pull request by eyelessrhyme7**
- fix division by 0 bugs in matrix when all tasks are <= currDate

closes #141

### CODE REVIEW

**Review by jack1e0 — APPROVED** [review]
lgtm

**Review by newway1814 — APPROVED** [review]
LGTM!

File: src/main/java/profplan/ui/MainWindow.java [hunk 1]
```diff
@@ -311,10 +310,13 @@ private void loadMatrix() {
...
-            System.out.println(split);
-            System.out.println(minDaysLeft);
-            System.out.println(maxDaysLeft);
+            long intermediate = (minDaysLeft + maxDaysLeft) / 10;
+            long split;
+            if (intermediate == 0) {
+              split = 1;
+            } else {
+              split = intermediate;
+            }
```
Comment (kiatkat) [inline src/main/java/profplan/ui/MainWindow.java:319]: Nice fix

**Review by kiatkat — APPROVED** [review]
LGTM!

**Review by newway1814 — APPROVED** [review]
LGTM!

**Pull request merged**